server/cli/start.go

```diff
@@ -230,6 +230,7 @@ func runQuit(cmd *cobra.Command, args []string) {
 var nodeCmds = []*cobra.Command{
 	initCmd,
 	startCmd,
+	certCmd,
```

Comment:
this is probably not right in that it implies `certCmd` cares about all kinds of flags that it really doesn't